bundles/org.openhab.persistence.jdbc/src/main/java/org/openhab/persistence/jdbc/internal/db/JdbcPostgresqlDAO.java

```diff
@@ -264,6 +264,7 @@ public void doStoreItemValue(Item item, State itemState, ItemVO vo, ZonedDateTim
     @Override
     protected String histItemFilterQueryProvider(FilterCriteria filter, int numberDecimalcount, String table,
             String simpleName, ZoneId timeZone) {
+        table = "\"" + table + "\"";
```

Comment:
I think it would be better to add the quotes when creating the query, i.e. in line 291 and 292. This way the quotes will not be included in the logging just below.

Reply:
I think it's important that the quotes are in the logging as well, otherwise you are logging an incorrect queries.

Reply:
We are not logging the query here, we are logging the parameters. Table name should be without quotes, it's only a technicality that we need to quote it in queries to preserve case.